nemo/collections/tts/models/fastpitch.py

```diff
@@ -178,13 +174,12 @@ def parser(self):
             ds_class_name = self._cfg.train_ds.dataset._target_.split(".")[-1]
 
             # TODO(Oktai15): remove it in 1.8.0 version
-            if ds_class_name == "AudioToCharWithPriorAndPitchDataset" or ds_class_name == "TTSDataset":
+            if ds_class_name == "TTSDataset":
                 self._parser = self.vocab.encode
             else:
                 raise ValueError(f"Unknown dataset class: {ds_class_name}")
         else:
             # TODO(Oktai15): remove it in 1.8.0 version
```

Comment:
Let's remove all the todo comments since you did them.

Reply:
Done